cassandra/roles/backup-cassandra/tests/test.yml

```diff
@@ -0,0 +1,5 @@
+---
+- hosts: localhost
+  remote_user: root
```

Comment:
What happens if we remove this? Not sure we need to be root for any of these commands.

Reply:
i removed remote_user: root in line 3 and things seem fine. The reason this was here was because it was automatically made when i made the skeleton using ansible-galaxy.